analytics/java/src/main/java/com/koverse/examples/analytics/SentimentAnalysis.java

```diff
@@ -82,7 +84,7 @@
      * words that appear in that message:
      */
 
-    UDF1 sentimentUDF = new UDF1<String, Double>() {
+    UDF1<String,Double> sentimentUDF = new UDF1<String, Double>() {
```

Comment:
I don't think you need <String,Double> twice ex. UDF1<String,Double> sentimentUDF = new UDF1<>()

Reply:
Actually Intellij screams if it's not there so I'm leaving it for now. It shouldn't be necessary but it doesn't hurt to have it.